src/bignum.h

```diff
@@ -127,7 +128,6 @@ class CBigNum
         return BN_get_word(self);
     }
 
-
```

Comment:
Mixing Coding style changes with the actual fixes is discouraged, we should open an specific issue for such